keyboards/kopibeng/mnk88/readme.md

```diff
@@ -0,0 +1,19 @@
+# MNK88
+
+![MNK88](https://i.imgur.com/QlKuV3w.png)
+
+A QMK-powered, VIA-enabled 88-key (F13) TKL PCB with support for ANSI/ISO layouts, split Backspace, split Right Shift and 6.25U/7U bottom row.
+
+* Keyboard Maintainer: kopibeng
+* Hardware Supported: A TKL keyboard with STM32F072 MCU
+* Hardware Availability: https://monokei.co/
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Updated, thanks.